services/ir/artifacts.mk

```diff
@@ -0,0 +1,20 @@
+# Get NeoFS LOCODE database
+
+LOCODE_DB_ARCHIVE_PATH=./vendor
+LOCODE_DB_ARCHIVE_FILE=locode_db.gz
+
+get.ir: LOCODE_DB_PATH?=
+get.ir:
+	@mkdir -p ${LOCODE_DB_ARCHIVE_PATH}
+
+ifeq (${LOCODE_DB_PATH},)
+	@echo "⇒ Download NeoFS LOCODE database from ${CHAIN_URL}"
```

Comment:
Looks like there should be `LOCODE_DB_URL` here

Reply:
Fixed.